### PR TITLE
Add proton-vpn-gtk-app

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

Resolves https://github.com/getsolus/packages/issues/981

**Test Plan**

Connect to protonvpn server, browse internet, check IP

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by malfisya** [comment]
Still missing `python-aiohttp` as rundeps . This is quite a rabbithole

**Review by EbonJaeger — COMMENTED** [review]
This looks like it was *fun* to sort out.

File: packages/p/proton-vpn-gtk-app/pspec_x86_64.xml [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            <Path fileType="library">/usr/lib/python3.11/site-packages/proton_vpn_gtk_app-4.3.3-py3.11.egg-info/dependency_links.txt</Path>
+            <Path fileType="library">/usr/lib/python3.11/site-packages/proton_vpn_gtk_app-4.3.3-py3.11.egg-info/entry_points.txt</Path>
+            <Path fileType="library">/usr/lib/python3.11/site-packages/proton_vpn_gtk_app-4.3.3-py3.11.egg-info/requires.txt</Path>
+            <Path fileType="library">/usr/lib/python3.11/site-packages/proton_vpn_gtk_app-4.3.3-py3.11.egg-info/top_level.txt</Path>
+            <Path fileType="data">/usr/share/applications/protonvpn-app.desktop</Path>
+            <Path fileType="data">/usr/share/icons/hicolor/36x36/apps/proton-vpn-logo.svg</Path>
+            <Path fileType="data">/usr/share/icons/hicolor/scalable/apps/proton-vpn-logo.svg</Path>
+            <Path fileType="data">/usr/share/metainfo/com.protonvpn.www.metainfo.xml</Path>
+            <Path fileType="data">/usr/version.py</Path>
+            <Path fileType="data">/usr/versions.yml</Path>
```
[the remark this answers is not in the document]
Comment (malfisya) [reply packages/p/proton-vpn-gtk-app/pspec_x86_64.xml:190]: So, the [version.py](https://b.opnxng.com/?9c7b54fe6ab64f1e#3skLNNeS4qcygtR4UGwcVvKJQFKsbtQLBsBqr4kiwzj8) has the logic to read the latest version from the [versions.yml](https://raw.githubusercontent.com/ProtonVPN/proton-vpn-gtk-app/v4.3.3/versions.yml) file. 

What do you think is best option? Moving these files to something like `/usr/share/proton-vpn-gtk-app` (which will only contain these two files)? Or just deleting it? Deleting these files does not cause error whatsoever.

**Comment by malfisya** [comment]
It was lot of _fun_. Also version 4.4.2 just released. lol. I will update it after this got merged.

**Comment by malfisya** [comment]
Turns out `python-aiohttp` and `python-yarl` was deprecated. So, I undeprecate it. The version of these package continuing its previous release number, hence checks failed.

File: packages/p/proton-vpn-gtk-app/pspec_x86_64.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (EbonJaeger) [reply packages/p/proton-vpn-gtk-app/pspec_x86_64.xml:190]: Is there any indication of what uses it? Does the application have a built-in changelog viewer?

[the remark this answers is not in the document]
Comment (malfisya) [reply packages/p/proton-vpn-gtk-app/pspec_x86_64.xml:190]: I think these files is meant for internal use to generate their changelog.md. By version 4.4.2, version.py is removed.

[the remark this answers is not in the document]
Comment (malfisya) [reply packages/p/proton-vpn-gtk-app/pspec_x86_64.xml:190]: I also was just made aware that version 4.4.2 is the official **beta** release. So, this is alpha level software.

[the remark this answers is not in the document]
Comment (EbonJaeger) [reply packages/p/proton-vpn-gtk-app/pspec_x86_64.xml:190]: Lovely. In that case, might as well just remove them.

**Comment by EbonJaeger** [comment]
I just noticed that we have `protonvpn-cli`, `protonvpn-gui`, and `protonvpn-nm-lib`. Does this replace any of those?

**Comment by malfisya** [comment]
It technically replace all those packages, as those are unmaintained upstream. But this is totally different app and AFAIK it does not cause conflict. Even if we decide to deprecate those packages, I don't think this package needs `replace` key.

**Comment by malfisya** [comment]
Thanks for the reminder @EbonJaeger . Turns out this app is in conflict with all those old app. I will tinker a bit more. Also in consideration, the old apps is still working as expected, while the new app just entered beta. There is some niceties in the new app but  I am not sure it is worth it to include this now. I will keep this drafted for now.

**Comment by malfisya** [comment]
A few changes:
- Updated to latest release (4.4.2)
- Add `python-proton-vpn-network-manager-wireguard` and `python-proton-vpn-killswitch-wireguard`
- Dropped `python-proton-vpn-killswitch` and `python-proton-vpn-connection` (Both of them are merged to `python-proton-vpn-core-api` by upstream)
- Now this will conflict with `proton-python-client`, `protonvpn-nm-lib`, `protonvpn-cli`, and `protonvpn-gui`

I think this is ready to be merged now.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**